### DISCUSSION TITLE
Category_id parameter in add_multibooking_form not working?

### DISCUSSION THREAD

**Discussion by Sp0run** [liedekef/events-made-easy#334] [General]
Hi Franky,

I can't seem to get the category_id parameter working in the add_multibooking_form short code. 

`[eme_add_multibooking_form category_id=1 template_id_header=8 template_id=9 template_id_footer=10]`

Am I missing something or is this a bug? I'm currently on version 2.4.12

Thanks

**Comment by liedekef** [comment]
```[eme_add_multibooking_form category_id=1 ]``` works fine here ...
Make sure the events are public

**Comment by Sp0run** [comment]
Yes, all events are public. But I've discovered that some of the events on this category are set to external events. External events must cause the shortcode to fail. I have re-assigned the categories on the external events and it's now working correctly.

**Reply by liedekef** [reply]
it's not the external part that is causing the "problem", but the fact that your external event probably does not have rsvp enabled.
This fixes that:
https://github.com/liedekef/events-made-easy/commit/c12324a4f9330a50f034892b4aa98a9ef05906f4

**Reply by Sp0run** [reply]
Thanks for your support 👍
